docs/user_manual.md

```diff
@@ -421,13 +421,22 @@ container.
 #### volumes
 Container volumes. Optional. 
 
-Specify either a single path to create a data volume, or a source path and a
-container path to mount a file or directory from the host. 
-  
-The container path can be suffixed with "rw" or "ro" to create a read-write or
-read-only volume, respectively.
+Specify a single path to create a data volume, a container path and a source
+path to mount a directory from the host, or a container path and a name to mount
+a named volume. The named volume will be created by docker if it does not exist;
+see [volumes][] in the Docker reference for full details. The container path can
+be suffixed with "rw" or "ro" to create a read-write or read-only volume mount,
+respectively.
+
+Format: `[container-path]:[rw|ro]:[host-path|volume-name]`.
```

Comment:
The format here doesn't match what is mentioned in the bullet points below, since the latter is representing it as a JSON object. Could you make this line match the below (i.e. with double quotes)?